webClient/src/app/editor/code-editor/monaco/monaco.component.scss

```diff
@@ -27,6 +27,11 @@
       background-color: yellow;
       color: black;
     }
+    ::ng-deep .myLineDecoration {

```

Comment:
I like the code here so far in the PR! Will continue scrolling down, but I think the UX could use a little work?

![image](https://user-images.githubusercontent.com/20528015/200693842-2b7460b4-5dc6-4d21-8236-537575d535c6.png)

It's pretty difficult to tell right now which line we are meant to start with especially in full-screen viewing (my thoughts: if the user doesn't know what line the URL takes them to, does the UX make it obvious which one they should be looking at?). I think something like this accomplishes it better maybe

![image](https://user-images.githubusercontent.com/20528015/200694928-47895d7f-1c35-429f-862c-caaff81d977b.png)

I also recolored it to make it distinctly different from a "selection"

`background: #e6e1ad;
 width: 50px !important;
 margin-left: -35px;`